src/main.rs

```diff
@@ -280,7 +280,10 @@ impl Fastmod {
         } else {
             println!("{}:{}-{}", path.to_string_lossy(), start_line, end_line);
         }
-        self.print_diff(old, new);
+        if !self.print_diff(old, new) {
+            println!("No changes.");
```

Comment:
`codemod` prints nothing in this case. Let's default to mirroring the behavior of `codemod`.

Reply:
Does codemod still print the filename/line number or do I need to reorganize things so we calculate the diff before we print anything?